docs/getting-started.md

```diff
@@ -173,10 +173,13 @@ For non-managed environments, see the following Kubernetes documentations:
 
 ## Customizing containerd
 
-containerd uses a configuration file located in `/etc/containerd/config.toml` for specifying daemon level options.
-A sample configuration file can be found [here](/docs/man/containerd-config.toml.5.md).
+containerd uses a configuration file located at `/etc/containerd/config.toml` to set daemon-level options.
 
-The default configuration can be generated via `containerd config default > /etc/containerd/config.toml`.
+Unlike some systems where you may override only the required values, containerd requires specifying the entire configuration file, even if most values are default.
```

Comment:
I'm not sure this is accurate, we don't require entire file to be specified. This is rather for convenience.

We do support imports, which means that we can load pieces of configuration and then we merge it into the final config struct: https://github.com/containerd/containerd/blob/f8b07365d260a69f22371964bb23cbcc73e23790/services/server/config/config.go#L295